Homework/Nam_Nguyen/Lesson11/LeetCode_76_MinimumWindowSubstring.java

```diff
@@ -0,0 +1,62 @@
+package Homework.Nam_Nguyen.Lesson11;
+
+import java.util.HashMap;
+import java.util.Map;
+
+public class LeetCode_76_MinimumWindowSubstring {
+
+    Map<Character, Integer> frequenciesOfStringT = new HashMap<>();
+    Map<Character, Integer> frequenciesOfStringS = new HashMap<>();
+
+    public String minWindow(String s, String t) {
```

Comment:
what is runtime of this algorithm?

Reply:
It is O(n + 2m*n) ~ O(m*n), where m = s.length, n = t.length